cli.js

```diff
@@ -95,9 +96,12 @@ getKeys((err, keys) => {
       xfwd: argv.xfwd,
       changeOrigin: argv['rewrite-origin'],
       autoRewrite: argv['rewrite-location'],
-      cookieDomainRewrite: argv['rewrite-cookies-domain'] ? '' : false
+      cookieDomainRewrite: argv['rewrite-cookies-domain'] ? '' : false,
+      headers: {
+        'Access-Control-Allow-Origin': '*'
```

Comment:
This must be configurable for people who do stuff with any security implications.  Preferably default to off and have a `--cors` flag to open it up.